scripts/new-watch.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+SRC="$1"
+DST="$2"
```

Comment:
It would be handy to have a safeguard, if the number of arguments isn't 2 otherwise it's very easy to copy meta- into meta- which is unfortunate. Also this could show a usage info